src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -120,9 +120,10 @@ class CollapsingNavigationMenu extends React.Component {
 
   componentDidUpdate() {
     const { isNewSelectedPath } = this.state;
-    const currentItemPosition = this.selectedItem && this.selectedItem.current ? this.selectedItem.current.getBoundingClientRect().bottom : null;
-    const navigationMenuHeight = document.querySelector('#terra-dev-site-nav-menu').getBoundingClientRect().bottom;
-    if (isNewSelectedPath && currentItemPosition > navigationMenuHeight) {
+    const currentItemPosition = this.selectedItem && this.selectedItem.current ? this.selectedItem.current.getBoundingClientRect() : null;
```

Comment:
you can use optional chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
```suggestion
    const currentItemPosition = this.selectedItem?.current ? this.selectedItem.current.getBoundingClientRect() : null;
```

Reply:
Updated here https://github.com/cerner/terra-dev-site/pull/261/commits/f7cc5a3c33dc0374e91fefb452b76a895a8e7543